testing/jupyterlab/test-info-pages/no-tab-trap-initial-page.md

```diff
@@ -0,0 +1,38 @@
+# No Tab Trap on Initial Page
+
+- id: 1wlgh6
+
+## Description
+
+When a user loads the app for the first time, they should be able to use the tab
+key to navigate without getting trapped.
+
+## Applicability
+
+Which apps does this test apply to.
+
+- JupyterLab
+
+## Accessibility Criteria
+
+This test is related to:
+
+- [WCAG Success Criterion 2.1.2 No Keyboard
+  Trap](https://www.w3.org/TR/WCAG22/#no-keyboard-trap)
+
+If this test fails, it means the app fails to conform to 2.1.2.
+
+If this test passes, however, it does not mean that the app conforms to 2.1.2.
+It only means that the app does not violate 2.1.2 for this specific test. This
+test can be combined with others to increase confidence that the app as a whole
+conforms to 2.1.2.
+
+## How to Verify the Test Manually
```

Comment:
I really like this section! Out of curiosity, are you basing this on any manual testing resource, or is this just a reflection of how you learned to test it?

Reply:
I'm not quite sure I understand what you're asking. Are you asking specifically about the technique I'm using to test for no tab traps (i.e., cycle back to the beginning) or are you asking about the way I wrote it up?

In either case, it's all my own work. I don't recall getting any of this from anywhere else.